newsPage.html

```diff
@@ -0,0 +1,567 @@
+                                        <h4>FOR FREE</h4>
+                                        <br>
+                                        <div class="full center">
+										    <a class="contact_bt" href="courses.html">SUBSCRIBE</a>
+										</div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- .pogoSlider -->
+            </div>
+        </div>
+    </div>
+    <!-- End Banner -->
+    <!--newsletter-->
+    <div style="text-align: center;">
+      <span class="theme_color" style="font-size:2em;">Update Yourself with buzzing Technology </span>
+    </div>
+    <div class="section margin-top_50">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6 layout_padding_2">
+            <div class="full">
+              <div class="heading_main text_align_left">
+                <h2><span>Welcome To</span> Newsletter</h2>
+              </div>
+              <div class="full">
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+                  irure dolor in reprehenderit in voluptate velit esse cillum
+                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                  cupidatat non proident, sunt in culpa qui officia deserunt
+                  mollit anim id est laborum
+                </p>
+              </div>
+              <div class="full">
+                <a class="hvr-radial-out button-theme" href="#">About more</a>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="full">
+              <div class="img__wrap">
+                <div class="full blog_img_popular  ">    
+                  <img class="img-responsive img__img" src="https://ebook3000.biz/wp-content/uploads/2018/01/Tech-Magazine-Issue-54-February-2018.jpg", alt="Newsletter">
+                  <h4 class="img__description"><a style="color:#fff;"href="#">Newsletter May</a></h4>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="section margin-top_50">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6 layout_padding_2">
+            <div class="full">
+              <div class="heading_main text_align_left">
+                <h2><span>Visit our</span> BLOG</h2>
+              </div>
+              <div class="full">
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
```

Comment:
Again, why have you kept this text. It's always good to fill this with relevant information.